_includes/nav.html

```diff
@@ -1,7 +1,7 @@
 <header class="container">
   <nav class="navbar navbar-expand-lg">
     <div class="container-fluid">
-      <a href="/" class="navbar-brand">
+      <a href="/" class="navbar-brand" aria-label="neovim logotype">
```

Comment:
Is there a specific reason "logotype" was chosen instead of "logo"?

Reply:
Done
Thanks